.github/actions/assemble-release/action.yml

```diff
@@ -6,7 +6,7 @@ inputs:
     default: GameData LICENSE* README* CHANGELOG*
   output-file-name:
     type: string
-    default: ${{ github.event.repository.name }}
+    default: ${{ github.event.repository.name || 'release'}}
```

Comment:
I don’t love that this creates a maintenance burden by requiring any changes to be done in several places.  Can’t this be set as a default env variable for the action?

Reply:
not sure what you're objecting to? the `|| 'release'` is just so it doesnt error when running in nektos-act where there is no repository.name

Reply:
Specifically that the same ` || /tmp/release` was in multiple locations